### PR TITLE
Icon and Rebrands

### PR DESCRIPTION
**Pull request by ranashreyas**
This pr contains changes specific to the DataHub rebranding, such as a new icon, and replacing "nbgitpuller link generator" with "DataHub Link Generator" wherever appropriate.

### CODE REVIEW

File: logo.png
Comment (ranashreyas) [inline logo.png:1]: New icon! let me know if this needs more adjustments. When in 32px format in the extensions bar, small graphical blemishes are not visible, so it is not an issue.

**Review by ranashreyas — COMMENTED** [review]
@balajialg and @ryanlovett

There are a few refactors of nbgitpuller for both chrome and firefox. This might take some extra time to check if there is a package/manifest error, but so far, none.

File: logo.png
Comment (ryanlovett) [reply logo.png:1]: I think the icon looks great! Thanks for taking on that challenge.

**Comment by ryanlovett** [comment]
This PR looks good to me. There are some instances of Datahub that I think need to be DataHub. @balajialg what do you think?

**Comment by balajialg** [comment]
@ryanlovett PR looks good to me too. Probably, @ranashreyas can fix the naming convention in the subsequent PRs?

@ranashreyas Good stuff with describing PRs. Super useful for reviewing.

**Pull request merged**